### PR TITLE
Serviceaccount automounttoken prevention

### PR DESCRIPTION
**Pull request by tommeramber**

### CODE REVIEW

File: open-policy-agent/authorization/serviceaccount_automounttoken_prevention/template.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sserviceaccountautomounttokenprevention
+  annotations:
+    description: Checks wheter the pod has a unique serviceAccount, correlated with the pod's name
```
Comment (igorkchyts) [inline open-policy-agent/authorization/serviceaccount_automounttoken_prevention/template.yaml:6]: @tommeramber 
The description does not fit the policy

File: open-policy-agent/authorization/serviceaccount_automounttoken_prevention/constraint.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sServiceAccountAutomountTokenPrevention
+metadata:
+  name: serviceaccount-automounttoken-prevention
+spec:
+  match:
```
Comment (igorkchyts) [inline open-policy-agent/authorization/serviceaccount_automounttoken_prevention/constraint.yaml:6]: @tommeramber @michaelkotelnikov 
Please add:
enforcementAction: dryrun

**Review by igorkchyts — COMMENTED** [review]
@tommeramber @michaelkotelnikov 
This policy if enforced will block default serviceaccounts (default,deployer,builder) from being created with new namespace/project. 

In my opinion, the default service accounts should be ignored, what do you think?

File: open-policy-agent/authorization/serviceaccount_automounttoken_prevention/template.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sserviceaccountautomounttokenprevention
```
Comment (igorkchyts) [inline open-policy-agent/authorization/serviceaccount_automounttoken_prevention/template.yaml:4]: @tommeramber @michaelkotelnikov 
The is very long, suggest to change to something shorter like 'k8snotokenautomount'

**Comment by michaelkotelnikov** [comment]
@igorkchyts Please add the fixes to the policy as you see fit, and commit them to @tommeramber 's fork.

**Review by igorkchyts — APPROVED** [review]

**Pull request merged**